### ISSUE TITLE
result of using maker.center.equalTo(CGPoing(x: X, y: Y)) is not base on the superview's original point

### ISSUE THREAD

**Issue by LiVincent-Zhang** [SnapKit/SnapKit#335]
### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform                | ios
 Platform Version        | 10.0
 SnapKit Version         | 3.0
 Integration Method      | carthage

### Issue Description

I found that if I used `maker.center.equalTo(CGPoing(x: X, y: Y))`, it generate a constraint based on the superview center, not the original point of superview. It strange because 

```swift
        self.snp.updateConstraints { (maker) in
            maker.center.equalTo(someCGPoint)
        }
```
equals to 

```swift
        self.snp.updateConstraints { (maker) in
            maker.center.equalTo(self.superview).offset(someCGPoint)
        }
```

May I ask why design this and can I commit a pull request to change this behavior?

**Comment by LiVincent-Zhang** [comment]
#336

**Comment by robertjpayne** [comment]
@LiVincent-Zhang good catch, in this case if the superview is implied it should probably be used as you expect.

**Comment by robertjpayne** [comment]
Fixed with [3.0.2](https://github.com/SnapKit/SnapKit/releases/tag/3.0.2)
